ludwig/backend/ray.py

```diff
@@ -210,6 +218,36 @@ def shutdown(self):
             ray.kill(handle)
         self.actor_handles.clear()
 
+    class BatchInferModel:
+        def __init__(
+                self,
+                remote_model: RayRemoteModel,
+                predictor_kwargs: Dict[str, Any],
+                output_columns: List[str],
+                features: List[Dict],
+                data_hdf5_fp: str,
+                *args, **kwargs
+        ):
+            self.model = remote_model.load()
+            self.predictor = Predictor(**predictor_kwargs)
+            self.output_columns = output_columns
+            self.features = features
+            self.data_hdf5_fp = data_hdf5_fp
+            self.batch_predict = partial(
```

Comment:
Is this being called? Looks like `self.predictor` is being used below.

Reply:
Good catch -- fixed now!